.github/workflows/build.yml

```diff
@@ -344,12 +344,79 @@ jobs:
       - name: Run make test_recursive for rust
         run: make -C lib/rs/test_recursive check
 
+  lib-python:
+    needs: compiler
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["2.x", "3.x"]
```

Comment:
I would drop Python v2. Python 2 has been sunsetted 1st of January 2020: https://www.python.org/doc/sunset-python-2/

Reply:
It would be nice to have a matrix to test against different versions of Python 3 as well:

https://github.com/apache/iceberg/blob/9dd00fab1dc160bb531091cbdda8f19ef5b3a32e/.github/workflows/python-ci.yml#L39-L42

This way you can also be more explicit in the supported Python versions (sometimes stuff breaks). Instead of:
https://github.com/apache/thrift/blob/1e3d90d8fd4160d538b7a4d902169eae5155e08a/lib/py/setup.py#L136

You can be more explicit in versions:
https://github.com/apache/airflow/blob/8d81963c014398a7ab14505fd8e27e432f1aaf5c/setup.cfg#L39-L42

Reply:
> I would drop Python v2. Python 2 has been sunsetted 1st of January 2020: https://www.python.org/doc/sunset-python-2/

feel free to review https://github.com/apache/thrift/pull/2588